crates/volta-core/src/error/kind.rs

```diff
@@ -1361,6 +1376,12 @@ to {}
                 "Could not write Path environment variable.
 
 Please ensure you have permissions to edit your environment variables."
+            ),
+            ErrorKind::Yarn2NotSupported => write!(
+                f,
+                "Yarn@2 is not recommended for use, and not supported by Volta.
+
+Please use Yarn@3 instead."
```

Comment:
My quick intuition here would be to make this slightly more verbose with writing out `version` as opposed to using the `@` syntax, since we're referring to `Yarn` capitalized, not the lower-case package name.

We also should probably make the CTA a little more future proof with something like `Please use version 3 or greater instead.`

Reply:
Good points. I updated this error message to be more verbose and future-proof in db9e6d1d0e1a2ca910231af66ce65b8f9416f382